benchmarks/fio_raid_baseline.sh

```
@@ -0,0 +1,99 @@
+    --ioengine=io_uring \
+    --iodepth=128 \
+    --direct=1 \
+    --group_reporting \
+    --time_based=0 \
+    --output-format=normal \
+    | tee -a "$RESULTS_FILE"
+
+echo "" | tee -a "$RESULTS_FILE"
+
+# Cleanup
+rm -rf "$TEST_DIR"/*
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to /* .

(SC2115)